doc/07-Preferences.md

```diff
@@ -3,34 +3,14 @@
 Preferences are settings a user can set for their account only,
 for example the language and time zone.
 
-Preferences can be stored either in INI files or in a MySQL or in a PostgreSQL database. By default, Icinga Web 2 stores
-preferences in INI files beneath Icinga Web 2's configuration directory.
-
-```
-/etc/icingaweb2/<username>/config.ini
-```
+Preferences can be stored either in a MySQL or in a PostgreSQL database. Database must be configured in the setup wizard.
```

Comment:
It's *a* database. So it'd be `The database must...`, though there is no requirement to do it in the wizard. The wizard wasn't mentioned previously, so I don't think it needs to be mentioned now.